src/routes/recipe.routes.ts

```diff
@@ -133,6 +133,46 @@ router.get(
   recipeController.readRecipeById
 );
 
+/**
```

Comment:
later on consider moving this documentation to separate files. This things stat po pile up quickly and make router a bit difficult to modify

Reply:
I would leave it as it is. This is JSDoc style where we're documenting some method directly above it